src/views/OnBoardingSubProduct/OnBoardingSubProduct.tsx

```diff
@@ -223,7 +223,6 @@ function OnBoardingSubProduct() {
         StepAddManager({
           product: subProduct,
           forward: () => {
```

Comment:
The forward action is not accepting the data of the step in order to store them (see onboarding to fix this)